compiler/docgen.nim

```diff
@@ -1034,13 +1063,16 @@ proc documentRaises*(cache: IdentCache; n: PNode) =
     if p5 != nil: n[pragmasPos].add p5
 
 proc generateDoc*(d: PDoc, n, orig: PNode, docFlags: DocFlags = kDefault) =
+  ## Goes through nim nodes recursively and collects doc comments.
+  ## Main function for `doc`:option: (aka `doc2`:option:) command,
```

Comment:
```suggestion
  ## Main function for `doc`:option: command,
```

(doc2 is now undocumented and all references should disappear; and even then, it shouldn't be re-defined here)